### ISSUE TITLE
make required signoffs page accept "product" url param

### ISSUE THREAD

**Issue by mozbhearsum** [mozilla-frontend-infra/balrog-ui#197]
It would be nice to be able to go directly to the required signoffs page for a given product, with something like /required_signoffs?product=Firefox

(Imported from https://bugzilla.mozilla.org/show_bug.cgi?id=1410419)

**Comment by helfi92** [comment]
That should be simple to implement. We already have a filter menu that we can use to add `?product=` in the URL.

**Comment by rbrishabh** [comment]
Hi @helfi92! I'll take this up!

**Comment by helfi92** [comment]
Go for it :)

**Comment by rbrishabh** [comment]
Hey @helfi92 ! If I'm not wrong, to test the changes I am making, I need both, balrog-ui and [balrog](https://github.com/mozilla/balrog) to be running on my machine. 

Otherwise, I will keep getting this error:
![image](https://user-images.githubusercontent.com/22501334/65137193-81a1fc80-da26-11e9-9c05-7e759d42e39c.png)


Is that right?

**Comment by helfi92** [comment]
Correct. Here's the balrog repo that you'll need to run in a separate terminal: https://github.com/mozilla/balrog.

**Comment by mozbhearsum** [comment]
@rbrishabh - If you have any issues getting the Balrog backend app running please let me know.

**Comment by rbrishabh** [comment]
Unfortunately, docker requires windows 10 pro to run on Windows. I have windows 10 home. @mozbhearsum, is there any other way I can run balrog backend app?

Thanks!

**Comment by rbrishabh** [comment]
I'll also give this a try with WSL. I will ask if I need any help.

**Comment by mozbhearsum** [comment]
> Unfortunately, docker requires windows 10 pro to run on Windows. I have windows 10 home. @mozbhearsum, is there any other way I can run balrog backend app?
> 
> Thanks!

Urgh, that is unfortunate! It's theoretically possible to run it without Docker, but there's no real instructions I can give you. Balrog depends on a MySQL database (which Docker provides), so you'd be looking at setting up your own database as well as figuring out how to get the backend itself running.

It looks like you may still be able to run Docker in Virtualbox mode though? At least, that's what this forum seems to suggest: https://forums.docker.com/t/installing-docker-on-windows-10-home/11722/2
